packages/hardware-trezor/src/transformers/toTrezorTxIn.ts

```diff
@@ -0,0 +1,18 @@
+import * as trezor from 'trezor-connect';
```

Comment:
Can you rename this file to `txIn.ts` because of consistency with `hardware-ledger` package